test/pop/blockchain/altblock_finalization_test.cpp

```diff
@@ -508,4 +508,29 @@ TEST_F(AltBlockFinalization2, FinalizeMaxBtcs) {
   assertTreesHaveNoOrphans(alttree);
 
   ASSERT_TRUE(alttree.setState(*tip, state));
+}
+
+TEST_F(AltBlockFinalization2, comparePopScore_test) {
+  altparam.mEndorsementSettlementInterval = 10;
+  altparam.mPreserveBlocksBehindFinal = 10;
+
+  // mine 10 altblocks
+  auto *forkPoint = mineAltBlocks(*alttree.getBestChain().tip(), 50);
```

Comment:
Looks like you mine 50 blocks not 10.
And what is this test case about? Make some intro.

Reply:
Fixed